### PR TITLE
refactor: use includes instead of or operators

### PR DESCRIPTION
**Pull request by WikiRik**
As mentioned in https://github.com/sequelize/sequelize/pull/13374

### CODE REVIEW

**Comment by sdepold** [comment]
Automated or manually fixed?

File: test/unit/model/define.test.js [hunk 1]
```diff
@@ -117,7 +117,7 @@ describe(Support.getTestDialectTeaser('Model'), () => {
           }
         });
 
-        if (dialect === 'postgres' || dialect === 'sqlite' || dialect === 'mssql') {
+        if (['postgres', 'mariadb', 'mssql'].includes(dialect)) {
```
Comment (sdepold) [inline test/unit/model/define.test.js:120]: should be pg, sqlite, mssql

Comment (sdepold) [reply test/unit/model/define.test.js:120]: since no test failed, I think we are lacking some

**Comment by WikiRik** [comment]
Mostly manually, only one file was done automatically since it had a lot of the same. I checked everything again before committing but it would be good if you could look at it as well

**Review by sdepold — CHANGES_REQUESTED** [review]

File: test/unit/model/define.test.js (hunk 1 above)
Comment (WikiRik) [reply test/unit/model/define.test.js:120]: Yup, will add some

**Comment by WikiRik** [comment]
Uhmm... I think we have a bigger issue; https://github.com/sequelize/sequelize/runs/4315490677?check_suite_focus=true

**Comment by WikiRik** [comment]
Draft until the tests are fixed on main

**Review by fncolon — APPROVED** [review]

File: test/unit/model/define.test.js (hunk 1 above)
Comment (sdepold) [reply test/unit/model/define.test.js]: Did you add a test?

Comment (sdepold) [reply test/unit/model/define.test.js]: > Yup, will add some

Actually ... it was a test already?!

**Pull request merged**